config/export_kc_config.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Variables
+source .env
```

Comment:
Consider using `source load_env.sh` instead. It is a wrapper with a mechanism that can override these env variables.